### PR TITLE
Fixes #34129 - HostDetails - UX Review - Spacing

### PR DESCRIPTION
**Pull request by Andrewgdewar**
The related Katello task is [here](https://github.com/Katello/katello/pull/9849)

This is adjusting spacing issues seen on the new host details UI.

Excerpt from the above task below: 

#### What are the changes introduced in this pull request?

- Adjusted spacing/placement on all pages across the new CV UI, as well as a few changes on the new Hosts UI.
 
#### Considerations taken when implementing this change?

- "Consistency" - Maria 

#### What are the testing steps for this pull request?

- Navigate through all new  CV/Hosts pages, using [this](https://marvelapp.com/prototype/8813483/screen/80289005)  design template as an outline for spacing.

- Note any pages that look off or don't follow a similar pattern.

### CODE REVIEW

File: webpack/assets/javascripts/react_app/components/HostDetails/index.js [hunk 1]
```diff
@@ -166,12 +161,7 @@ const HostDetails = ({
             tabs={tabs}
             router={history}
           >
-            <Tabs
-              style={{
-                width: window.innerWidth - (isNavCollapsed ? 95 : 220),
-              }}
-              activeKey={activeTab}
-            >
+            <Tabs activeKey={activeTab} className="host-details-tabs">
```
Comment (amirfefer) [inline webpack/assets/javascripts/react_app/components/HostDetails/index.js]: The dynamic width is calculated for having the tabs scrolling mechanism work as expected when the menu is collapsed, removing it causes a regression:

before:
https://user-images.githubusercontent.com/11807069/146046854-d3036cec-dbac-4cf0-927b-c6650ff06dfb.mov

with this PR:
https://user-images.githubusercontent.com/11807069/146046896-9a2f99a4-026c-447c-ad7d-259825536c37.mov

**Review by amirfefer — DISMISSED** [review]
Thanks @Andrewgdewar  for this!
Overall looks good 👍 

Just one regression that I've found, plus  we should fix ansible's secondary tabs spacing as well:
Screen Shot 2021-12-14 at 18 40 26

File: webpack/assets/javascripts/react_app/components/HostDetails/HostDetails.scss [hunk 2]
```diff
@@ -6,30 +6,48 @@
...
-  max-width: 60%;
-  margin-right: 8px;
-}
+  .host-details-tabs {
+    margin: 0 24px;
+  }
+
+  .host-details-tab-item {
+    padding: 16px 24px;
+    background: #f0f0f0;
```
Comment (amirfefer) [inline webpack/assets/javascripts/react_app/components/HostDetails/HostDetails.scss]: for colors we can use pf4 variables:
```scss
@import '~@theforeman/vendor/scss/variables';

.host-details-tab-item {
  background-color: $pf-color-white;
```

File: webpack/assets/javascripts/react_app/components/HostDetails/index.js (hunk 1 above)
Comment (Andrewgdewar) [reply webpack/assets/javascripts/react_app/components/HostDetails/index.js]: Fixed

File: webpack/assets/javascripts/react_app/components/HostDetails/HostDetails.scss (hunk 2 above)
Comment (Andrewgdewar) [reply webpack/assets/javascripts/react_app/components/HostDetails/HostDetails.scss]: I prefer var() imports, done.

File: webpack/assets/javascripts/react_app/components/HostDetails/index.js [hunk 3]
```diff
@@ -53,7 +53,6 @@ const HostDetails = ({
     `/api/hosts/${id}`,
     HOST_DETAILS_API_OPTIONS
   );
-
   const isNavCollapsed = useSelector(selectIsCollapsed);
```
Comment (ezr-ondrej) [inline webpack/assets/javascripts/react_app/components/HostDetails/index.js:56]: I'm bit confused, you've removed it previously and it worked, why is it now needed again?

Comment (Andrewgdewar) [reply webpack/assets/javascripts/react_app/components/HostDetails/index.js:56]: I added it back to satisfy when the tabs component needs the horizontal scroll functionality.
I need to know if this is true or false for the correct pixel change to be applied on said component.

Comment (ezr-ondrej) [reply webpack/assets/javascripts/react_app/components/HostDetails/index.js:56]: oh, right :) get it :+1: and I liked that we would get rid of it :( xD

**Review by ezr-ondrej — APPROVED** [review]
Works for me :+1:
Thanks @Andrewgdewar !

**Comment by ezr-ondrej** [comment]
Thanks @amirfefer for the review, both of your concerns were addressed, merging now :)

**Comment by Andrewgdewar** [comment]
Cheers guys

**Comment by amirfefer** [comment]
CP - https://github.com/theforeman/foreman/pull/9003

**Pull request merged**